message_ix_models/model/material/report/reporting.py

```diff
@@ -179,7 +180,7 @@ def report(context, scenario):  # noqa: C901
         nodes.remove("R12_GLB*")
 
     # Path for materials reporting output
-    directory = context.get_local_path("report", "materials")
+    directory = package_data_path("material", "reporting_output")
```

Comment:
I sense this change has been made because we haven't provided good enough documentation and examples on how to ensure output files are placed in a desired location.

In general, code should only be outputting to directories within the package itself if it's intended that the generated files will be committed to the repo. Otherwise, the user should choose and be able to configure a local directory for output files, or the code should pick a sensible default.

Let's not hold up this PR to revert, but have a conversation to address the usability issue and then revert later.